packages/components/psammead-radio-schedule/src/ProgramCard/__snapshots__/index.test.jsx.snap

```diff
@@ -533,6 +534,7 @@ exports[`ProgramCard should render correctly for next 1`] = `
     </span>
     <time
       class="c10"
+      datetime="PT30M"
```

Comment:
Confirm this displays the expected duration (`30:00`) in IE.

Reply:
Do you mean to confirm if the `datetime` value changes to `30:00` instead of `PT30M` in IE? If yes, this isn't supposed to change because that is a valid `datetime` format for the `time` tag